tests/reporting/test_reporting_windows.py

```diff
@@ -41,9 +41,58 @@ def test_reporting_window_initialize(localFixture, chain, mockUniverse, mockPart
     previousReportingWindow = localFixture.upload('../source/contracts/reporting/ReportingWindow.sol', 'previousReportingWindow')
     mockUniverse.setReportingWindowByTimestamp(previousReportingWindow.address)
     assert reportingWindow.getPreviousReportingWindow() == previousReportingWindow.address
-    chain.head_state.timestamp = dispute_end_time + 1
+    
+    chain.head_state.timestamp = reportingWindow.getStartTime() - 1
+    assert reportingWindow.isOver() == False
+    assert reportingWindow.isActive() == False
+    assert reportingWindow.isReportingActive() == False
+    assert reportingWindow.isDisputeActive() == False
+
+    chain.head_state.timestamp = reportingWindow.getStartTime()
+    assert reportingWindow.isActive() == False
+    assert reportingWindow.isReportingActive() == False
+    assert reportingWindow.isDisputeActive() == False
+
+    chain.head_state.timestamp = reportingWindow.getStartTime() + 1
+    assert reportingWindow.isActive() == True
+    assert reportingWindow.isReportingActive() == True
+    assert reportingWindow.isDisputeActive() == False
+
+    chain.head_state.timestamp = reportingWindow.getReportingEndTime() 
+    assert reportingWindow.isActive() == True
+    assert reportingWindow.isReportingActive() == False
+    assert reportingWindow.isDisputeActive() == False
+
+    chain.head_state.timestamp = reportingWindow.getReportingEndTime() + 1
+    assert reportingWindow.isActive() == True
+    assert reportingWindow.isReportingActive() == False
+    assert reportingWindow.isDisputeActive() == True
+    
+    chain.head_state.timestamp = reportingWindow.getDisputeStartTime()
+    assert reportingWindow.isActive() == True
+    assert reportingWindow.isReportingActive() == False
+    assert reportingWindow.isDisputeActive() == False
+    assert reportingWindow.isOver() == False
+
+    chain.head_state.timestamp = reportingWindow.getDisputeStartTime() + 1
+    assert reportingWindow.isActive() == True
+    assert reportingWindow.isReportingActive() == False
+    assert reportingWindow.isDisputeActive() == True
+    assert reportingWindow.isOver() == False
+    
+    chain.head_state.timestamp = reportingWindow.getEndTime() 
+    assert reportingWindow.isActive() == False
+    assert reportingWindow.isReportingActive() == False
+    assert reportingWindow.isDisputeActive() == False
     assert reportingWindow.isOver()
 
+    chain.head_state.timestamp = reportingWindow.getEndTime() + 1
+    assert reportingWindow.isActive() == False
+    assert reportingWindow.isReportingActive() == False
+    assert reportingWindow.isDisputeActive() == False
+    assert reportingWindow.isOver()
```

Comment:
For consistency, recommend adding `== True` to this.  Alternatively, make all of the others be `assert not ...` and `assert ...`.

Reply:
update to add "== True" for consistency.